index.html

```diff
@@ -405,40 +433,41 @@ <h2 id="format">The did:orb Format</h2>
 
         <pre>
           did-orb-format        = "did:orb:" (orb-canonical-did / orb-scheme-did / unpublished-did)
-          orb-canonical-did     = min-graph-cid ":" did-suffix
-          orb-scheme-did        = (dht-scheme / webcas-scheme) ":" orb-canonical-did
-          unpublished-did       = "interim:" reg-name ":" did-suffix [":" long-form-suffix-data]
-          dht-scheme            = ( "ipfs" )
-          webcas-scheme         = "webcas:" reg-name
+          orb-canonical-did     = min-graph-multihash ":" did-suffix
+          orb-scheme-did        = (hashlink-scheme / hostmeta-scheme)
+          hashlink-scheme       = "hl:" min-graph-multihash ":" graph-hl-metadata ":" did-suffix
```

Comment:
Our existing resolver logic works by fetching a CAS file and then querying hostmeta that is embedded into the anchor origin.

This commit effectively encodes what we are doing for discovery into the DID string:
- Supply a link to the immutable file containing the anchor origin.
- Directly supply the information needed to find the hostmeta file (via https or ipns).

Reply:
The hashlink to an anchor event also matches up with https://github.com/trustbloc/activityanchors/pull/6.